### PR TITLE
AZP: add new OS for release CI

### PR DESCRIPTION
**Pull request by avildema**
## Why
Expand the matrix of os/cuda versions for binary release packages

### CODE REVIEW

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@avildema pls port to v1.10 as well

**Comment by avildema** [comment]
bot:pipe:retest

**Comment by shamisp** [comment]
👍

File: buildlib/azure-pipelines-release.yml [hunk 1]
```diff
@@ -16,8 +16,10 @@ resources:
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5.0-cuda10.1:1
     - container: centos7_cuda10_2
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5.0-cuda10.2:1
-    - container: centos7_cuda11_0
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5.0-cuda11.0:1
+    - container: centos7_cuda11_1
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5.x-cuda11.1:1
```
Comment (yosefe) [inline buildlib/azure-pipelines-release.yml:20]: the container name should be mofed-5.0
the artifact name should be mofed-5.x

Comment (avildema) [reply buildlib/azure-pipelines-release.yml:20]: Are you seriously ? need to rebuild all containers (will duplicate)  , and this name already use in master CI 
https://github.com/openucx/ucx/blob/master/buildlib/azure-pipelines-pr.yml#L12

**Comment by avildema** [comment]
bot:pipe:retest

File: buildlib/azure-pipelines-release.yml (hunk 1 above)
Comment (yosefe) [reply buildlib/azure-pipelines-release.yml:20]: sorry I didn't notice this before, but the container name should be the exact mofed version
we can keep it for now. but for next containers use right name, and find a good opportunity to fix existing ones

**Review by yosefe — APPROVED** [review]

**Pull request merged**